playbook/roles/holms.balancer/defaults/main.yml

```diff
@@ -8,6 +8,28 @@ balancer_ssl_pem: ""
 balancer_ssl_trusted_crt: ""
 balancer_ssl_trusted_key: ""
 
-listen: "80 http2"
+# Cusomization
+# See innards at playbook/roles/holms.balancer/templates/vhosts.j2
+listen: "443 ssl"
 server_name: "myserver1.com"
-error_page: ""
+# A redirect with specific port uses error_page and port
+# error_page 497 301 =307 https://$host:{{ port }}$request_uri
+error_page: true
+port: 9000
+# Redirect without a port uses
+server_without_port: false
+
+# For example.
+# A rediraction without a specific port, SSL only. Set
```

Comment:
There is a bunch of typos and weirdly phrased sentences. Please ask @kamilongus to proofread and possibly suggest better variable names, since a) his written English is better and b) he'll also have to use this module so he has to understand what each of the params stands for.